IntegrationTests/common/script/CompareMemPrintsFW.py

```diff
@@ -123,48 +114,26 @@ def compare(comparison_filename="", fail_on_error=False, file_location='./', pre
             reference_type = ReferenceType[reference_type_string]
         print("Comparing " + reference_type.FullName() + " values ... ")
 
-        # Find the layer names for the reference file
-        if reference_type == ReferenceType.FM:
-            layers = re.search("(L[0-9]L[0-9])",reference_filename).group(0)
-            print("Comparing the values for layers "+str(layers)+" to the reference file "+str(reference_filename)+" ... ")
```

Comment:
I found it useful to have the reference filename printed out. Could we keep that, and actually print it out for all memory types, not just FullMatches?

Reply:
Fixed. It now prints out a message such as "Comparing TB results to ref. file ../../../emData/MemPrints/TrackletProjections/AllProj_AP_L3PHIC_04.dat ..." before each comparison with a new file.